blocks/footer/footer.css

```diff
@@ -1,22 +1,103 @@
-footer {
-  background-color: var(--spectrum-gray-50);
-  font-size: var(--spectrum-body-size-xs);
+:root {
+  --color-background-footer: var(--spectrum-white);
+
+  &:has(#color-scheme option[value="dark"]:checked) {
+    --color-background-footer: var(--spectrum-gray-50);
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
```

Comment:
We've applied the anti-aliasing properties on the `body` element in `base.css`, is stating it here redundant?

Reply:
it sure is